tests/unit/returners/test_local_cache.py

```diff
@@ -72,6 +72,12 @@ def test_clean_old_jobs_empty_jid_dir_removed(self):
         # Make sure there are no files in the directory before continuing
         self.assertEqual(jid_file, None)
 
+        # Windows needs some time to release the file handles to the new temp
+        # dir before trying to clean the job cache to avoid a race condition
```

Comment:
If any call in that function isn't properly cleaning up a filehandle, that's a serious bug. We need to investigate why and solve that issue.